common/src/Allocator.h

```diff
@@ -140,9 +140,9 @@ class Allocator {
         
         if (chunk->full())
             fullChunks().push_back(chunk);
-            else
```

Comment:
Feel free to use braces even for one liner if / else blocks. I changed my opinion about this and now prefer to use braces in every case.